### PR TITLE
update to test_json

### PR DESCRIPTION
**Pull request by dannmorr**
update with county child and zip child jsons

### CODE REVIEW

File: test_jsons.py [hunk 1]
```diff
@@ -15,6 +17,17 @@ def test_zip_poverty():
...
+    #runs county script to create/update county json
+    county_child_poverty_script()
+    #tests county json file
+    check_file('final_jsons/acs5county_child_poverty_output.json')
+    
+def test_zip_child_poverty():
+    #runs county script to create/update county json
+    zip_child_poverty_script()
+    #tests county json file
+    check_file('final_jsons/acs5zip_child_poverty_output.json')
```
Comment (michaelpkuhn) [inline test_jsons.py:31]: Could you merge this into your other PR?

Comment (michaelpkuhn) [reply test_jsons.py:31]: Not just that portion in particular. We can do one PR

Comment (dannmorr) [reply test_jsons.py:31]: Can they not be merged separately?

Comment (michaelpkuhn) [reply test_jsons.py:31]: They can but if they conflict, that conflict will need to be resolved before the other merges. If it is in one PR, then we know it does not conflict.

**Pull request closed without merging**